libjulia/optimiser/FullInliner.cpp

```diff
@@ -156,6 +156,25 @@ void InlineModifier::visit(Expression& _expression)
 	m_statementsToPrefix.emplace_back(BodyCopier(m_nameDispenser, fun.name + "_", variableReplacements)(fun.body));
 }
 
+void InlineModifier::visit(Statement& _statement)
+{
+	ASTModifier::visit(_statement);
+	// Replace pop(0) expression statemets by empty blocks.
```

Comment:
I think this is not replaced by the new simplification rules?

Reply:
No, the simplification rules only operate on expression level.